vtexdocs/devportal
# Update/dropdown item descriptions

#### What is the purpose of this pull request?

Update the sections descriptions on the header's dropdown.

#### What problem is this solving?

The descriptions in the dropdown are different from the documentation cards in Landing Page, although they needed to be the same. As the dropdown descriptions will be the same as the cards description, I will remove unnecessary objects and fix the types of the elements.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-53--elated-hoover-5c29bf.netlify.app/) and check if:
- The descriptions of the sections in the dropdown (header) are the same as the Documentation cards in the landing page.
- If the style of the dropdown elements has not changed.
- If the style of the see also section elements has not changed.

#### Screenshots or example usage

| before | after |
| -------- | -------- |
|![image](https://user-images.githubusercontent.com/42784961/176672066-ccd25158-e058-4b47-bba4-7d226daf02a4.png)|![image](https://user-images.githubusercontent.com/42784961/176672216-76da3273-2138-4120-b293-312082e42229.png)|

#### Types of changes
